docs/UserGuide.adoc

```diff
@@ -34,7 +34,7 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
 * *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
```

Comment:
While I appreciate the work done, there is no need for you to edit the User Guide as I have drafted a User Guide and made a PR.